openshift_metrics/invoice.py

```diff
@@ -9,20 +9,23 @@
 GPU_A100 = "NVIDIA-A100-40GB"
 GPU_A100_SXM4 = "NVIDIA-A100-SXM4-40GB"
 GPU_V100 = "Tesla-V100-PCIE-32GB"
+GPU_H100 = "???-H100-???-???" # TODO Confirm name of H100 in Openstack
```

Comment:
@joachimweyl @naved001 @knikolla What will the name of the H100 resource be in Openstack?

Reply:
@QuanMPhm the name is `NVIDIA-H100-80GB-HBM3`

```
➜  ~ oc get nodes wrk-6 -o yaml | yq -r  '.metadata.labels."nvidia.com/gpu.product"'
NVIDIA-H100-80GB-HBM3
```